tests/test_activate.sh

```diff
@@ -10,7 +10,16 @@ rm -rf ${TESTENV}
 
 echo "$0: Creating virtualenv ${TESTENV}..." 1>&2
 
-${VIRTUALENV} ${TESTENV} | tee ${ROOT}/tests/test_activate_output.actual
+PY_BLEND=$(python -c "import sys; print('pypy' if '__pypy__' in sys.builtin_module_names else 'python');")
+cat >${ROOT}/tests/test_activate_output.expected <<EOL
+New ${PY_BLEND} executable in /tmp/test_virtualenv_activate.venv/bin/python
+Installing setuptools, pip, wheel...done.
+EOL
+
+# "Path not in prefix" warning is not really an error, being expected under pypy
+# > Path not in prefix '/home/travis/virtualenv/pypy2.7-5.8.0/include' '/opt/python/pypy2.7-5.8.0'
+
```

Comment:
How about checking venv creation exit code?

Reply:
@krosinski I do agree with you that checking the stdout/stderr output makes no sense, that's not really an API. 

I would check for behaviour which means: exit code PLUS validating that the environment was activated, which to me it means checking the `which python` returns the venv version of python.

Overall test_activate.sh is a mess and last week I started to rewrite it as pytest unittests and also to test it for all 4 supported shells:(bash, csh, fish and powershell. You can see the WIP work at https://github.com/pypa/virtualenv/pull/1090 but it will take some time to get it ready as increasing the number of shells multiplied by number of supported platforms and python versions does uncover other bugs.

I think it will be better if we first mere a gradual improvement in test_activate.sh because otherwise we will block testing other fixes. I would prefer to avoid the case where a PR takes so much time that it never reaches the merge status.

If you are OK, I would not mind removing the output testing.